src/settings/Settings.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import SelectWithLabelFormGroup from '../components/input/SelectWithLableFormGroup'
+import i18n, { resources } from '../i18n'
+import useTitle from '../page-header/useTitle'
+
+const Settings = () => {
+  const { t } = useTranslation()
+  useTitle(t('settings.label'))
```

Comment:
let's add a test to make sure that the title gets rendered properly

Reply:
✅ in 494dc2e